packages/itwinui-react/src/core/utils/hooks/useStyles.ts

```diff
@@ -25,12 +25,17 @@ export const useStyles = (options?: {
   document?: () => Document | undefined;
 }) => {
   const context = _React.useContext(ThemeContext);
-  const loaded = _React.useRef(false);
-
-  const includeCss = options?.includeCss ??
-    context?.includeCss ?? { withLayer: true };
 
   useIsomorphicInsertionEffect(() => {
+    if (!context) {
+      return;
+    }
+
+    const loaded = context.stylesLoaded;
```

Comment:
Storybook does not load styles because it does not have context.

Maybe we could define loaded outside isomorphic effect and not do early return when context is not provided?
```js
  const loaded = !!context ? context.stylesLoaded : _React.useRef(false);
```

Or we could fix storybook styling.

Reply:
looks like v3 storybook has context but v2 one doesn't.

in v2 we support using our components without ThemeProvider, so i guess we'll need to support usage without context.

added fallback in [`b1ff2e7`](https://github.com/iTwin/iTwinUI/pull/1625/commits/b1ff2e7e5afd346d5cde2ecaef02bd60f5978e50)